### PR TITLE
Add tier as a ClusterCreate parameter

### PR DESCRIPTION
**Pull request by ahosnyms**
The content to be added to Changelog is as follows:
* 
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ahosnyms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ahosny). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: lib/commands/arm/hdinsight/hdiCustomization._js [hunk 1]
```diff
@@ -343,7 +343,8 @@ __.extend(HdiCustomization.prototype, {
           clusterType: clusterCreateParameters.clusterType
         },
         clusterVersion: clusterCreateParameters.version,
-        operatingSystemType: clusterCreateParameters.osType
+        operatingSystemType: clusterCreateParameters.osType,
+		clusterTier: clusterCreateParameters.clusterTier
```
Comment (amarzavery) [inline lib/commands/arm/hdinsight/hdiCustomization._js]: fix tabbing. IT should be 2 spaces for node.js

File: test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js [hunk 2]
```diff
@@ -197,7 +251,7 @@ describe('arm', function() {
             done();
           }
         });
-      });
+      });
```
Comment (amarzavery) [inline test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js]: please do not have extra spaces

**Comment by amarzavery** [comment]
@ahosnyms - The CI is failing. Please make sure that the CI passes and also make changes as per the other comments in the review.

**Comment by ahosnyms** [comment]
CI should be fixed now, also other comments done. I added my github to TravisCI, but no build kicked off when pushed a commit, is there any missing step?

**Pull request merged**